demos/typography.html

```diff
@@ -84,16 +84,9 @@ <h6 class="mdc-typography--subtitle2">Subtitle 2</h6>
 
       <h6 class="mdc-typography--headline6 demo-typography--heading-baseline">Baseline</h6>
       <section class="demo-typography--section-baseline">
-        <h2 class="demo-typography__title mdc-typography--headline6">
-          Our Changing Planet
-        </h2>
-        <h3 class="demo-typography__subtitle mdc-typography--subtitle2">
-          by Kurt Wagner
-        </h3>
-        <div class="demo-typography__body mdc-typography--body2">
-          Visit ten places on our planet that are undergoing the biggest changes today.
-        </div>
-
+        <h2 class="demo-typography__title mdc-typography--headline6">Our Changing Planet</h2>
+        <h3 class="demo-typography__subtitle mdc-typography--subtitle2">by Kurt Wagner</h3>
+        <div class="demo-typography__body mdc-typography--body2">Visit ten places on our planet that are undergoing the biggest changes today.</div>
         <line class="demo-typography-line-1">+34px</line>
```

Comment:
Why are we using a non-standard `<line>` element instead of a `<div>` or something?